package.json

```diff
@@ -77,6 +77,8 @@
     "test:e2e:watch": "vitest watch tests",
     "test:all": "vitest run",
     "test:watch": "vitest watch",
+    "test:examples": "vitest --max-concurrency=10 --config ./examples/vitest.examples.config.ts run tests/examples",
+    "test:examples:watch": "vitest --max-concurrency=10 --config ./examples/vitest.examples.config.ts run tests/examples",
```

Comment:
Instead of passing `max-concurrency` as a flag, you can set it directly within the configuration file `vitest.examples.config.ts`